src/agent/ebpf/bcc/bcc.go

```diff
@@ -83,12 +83,16 @@ func (m *module) attachKProbe(probe *ebpfpb.ProbeSpec) error {
 	syscallName := bcc.GetSyscallFnName(probe.Target)
 	if probe.Entry != "" {
 		if err := m.attachKEntryProbe(syscallName, probe.Entry); err != nil {
-			return err
+			if err := m.attachKEntryProbe(probe.Target, probe.Entry); err != nil {
```

Comment:
Using filed prefixes to distinguish the type looks a little magic, It will increase the use of the user cost. because this is the proprietary method in starship. But it seems we do not have any better solution. So as you wish.

Reply:
I will update doc in same time.